File: swagger/models/update_contact_log_comment.py
from .base_model import BaseModel


class UpdateContactLogComment(BaseModel):
	"""
	no description available

	Attributes:

		id
		The ID of the contact log comment to be updated. Pass a value of `0`
		to add a new comment to the contact log.

		text
		The new text for the comment.
	"""

	input_map = {
		'Id': 'id',
		'Text': 'text',
		}

	output_map = {
		'id': 'Id',
		'text': 'Text',
		}

File: swagger/Models/UpdateContactLogComment.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class UpdateContactLogComment extends BaseModel
{
	/**
	 * The ID of the contact log comment to be updated. Pass a value of `0`
	 * to add a new comment to the contact log.
	 * 
	 * @var integer
	 */
	public $id;
	/**
	 * The new text for the comment.
	 * 
	 * @var string
	 */
	public $text;
}
